S7.Net/PLC.cs

```diff
@@ -15,6 +15,9 @@ public partial class Plc : IDisposable
         private TcpClient tcpClient;
         private NetworkStream stream;
 
+        private int _readTimeout = System.Threading.Timeout.Infinite;
```

Comment:
Please adhere to the naming style already used in this file (i.e., name it `readTimeout`)